src/plone/recipe/zope2instance/recipe.py

```diff
@@ -922,13 +926,30 @@ def build_wsgi_ini(self):
             except OSError:
                 raise
 
+        # Load logging template or use default
+        global logging_ini_template
+        logging_ini_template_path = self.options.get("logging-ini-template")
```

Comment:
```suggestion
        wsgi_logging_ini_template_path = self.options.get("wsgi-logging-ini-template")
```

I would be more specific.